HumanPlayer.js

```diff
@@ -0,0 +1,22 @@
+const allTheWords = require('./word.js')
+
+class HumanPlayer{
+    constructor(human){
+        this.human = human
+        this.word = allTheWords[(Math.floor(Math.random() * dictionary.length))]
```

Comment:
Instead of `dictionary.length` It should be the length of the array of words you're using, this being `allTheWords`